### PR TITLE
add "no-throw-literal" rule

### PR DESCRIPTION
**Pull request by g-plane**
Ref: https://eslint.org/docs/rules/no-throw-literal

### CODE REVIEW

File: src/rules/no_throw_literal.rs [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  use super::*;
+  use crate::test_util::test_lint;
+  use serde_json::json;
+
+  #[test]
+  fn check_literal() {
+    test_lint(
+      "no_throw_literal",
+      "throw 'kumiko'",
+      vec![NoThrowLiteral::new()],
```
Comment (bartlomieju) [inline src/rules/no_throw_literal.rs:63]: Can you add a few more tests cases for bad usage of throw?

Comment (g-plane) [reply src/rules/no_throw_literal.rs:63]: Bad usage? Such as other types of literals?

Comment (bartlomieju) [reply src/rules/no_throw_literal.rs:63]: Yes, a number, null, etc.

Comment (g-plane) [reply src/rules/no_throw_literal.rs:63]: Done.

**Pull request merged**